src/shared/varlink-io.systemd.Machine.c

```diff
@@ -72,6 +77,8 @@ SD_VARLINK_DEFINE_INTERFACE(
                 &vl_type_Timestamp,
                 &vl_method_Register,
                 &vl_method_Unregister,
+                SD_VARLINK_SYMBOL_COMMENT("Terminate virtual machine, killing its processes"),
```

Comment:
"virtual machine" => "machine" since machined manages both virtual machines and containers